src/scss/components/_feature.scss

```diff
@@ -0,0 +1,18 @@
+.feature {
+  margin: 2rem auto;
+  border-radius: 3px;
+  width: 35rem;
+  display: grid;
+  grid-template-rows: min-content min-content min-content;
+  &__image {
+    width: 100%;
```

Comment:
According to the design file (https://github.com/custonote/custonote-info/blob/development/Images/landing%20page%20version%202.jpg), the image should be 50-60% of the width of the container while being centered